credentialdigger/client.py

```diff
@@ -1378,3 +1381,64 @@ def update_similar_snippets(self,
                 self.update_discovery(d['id'], state)
                 n_updated_snippets += 1
         return n_updated_snippets
+
+    def assign_categories(self, discoveries):
+        """ Add category to each discovery
+
+        Parameters
+        ----------
+        discoveries: list
+            List of discoveries without assigned categories to them
+
+        """
+        rulesdict = self.get_rules()
+        for discovery in discoveries:
+            if discovery['rule_id']:
+                category = rulesdict[discovery['rule_id'] - 1]['category']
+                discovery['category'] = category
+            else:
+                discovery['category'] = '(rule deleted)'
+
+    def export_discoveries_csv(self, repo_url, states='all'):
```

Comment:
why should we add this method here while it is introduced only for a CLI?
Can't you implement it in the CLI?

Reply:
This function has been added here because I am willing to use it later on for the 'export_discoveries' feature requested by this issue here: https://github.com/SAP/credential-digger/issues/114

I wanted to to use one export function for these two different features, and I found the Client to be the best place to contain such a function.